GA-worksheet.md

```diff
@@ -6,18 +6,18 @@
 
 
 ##Project #2 Part #1: Header HTML
-1. What is the purpose of the header tag?
+1. What is the purpose of the header tag? To put headers inside of. It's for headers.
 
-2. What is a Navigation Bar?
+2. What is a Navigation Bar? A navigation bar is like a "portal" to a list of links, to help users of the website access           specific information easier.
 
-3. What tag do we use to make a navigation bar? <      >
+3. What tag do we use to make a navigation bar? you use a { } or use <ul> and <li>.
 
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for __unordered list__
+    <li> Stands for ___list___
 
-5. What does the <a> tag do?
+5. What does the <a> tag do? it explicitly sets a value, or values for a link since links don't inherit values from their parent element.
```

Comment:
Simply put, it defines a link to somewhere, with the destination defined in the attribute href.